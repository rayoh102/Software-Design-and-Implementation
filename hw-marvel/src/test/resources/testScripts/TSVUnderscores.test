# Tests if underscore inputs are properly converted to spaces

# Load Graph
LoadGraph custom sampleData.tsv

# Find a short path
FindPath custom Hero6 Hero3