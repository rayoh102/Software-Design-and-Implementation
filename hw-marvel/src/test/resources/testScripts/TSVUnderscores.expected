# Tests if underscore inputs are properly converted to spaces

# Load Graph
loaded Graph custom

# Find a short path
path from Hero6 to Hero3:
Hero6 to Hero3 via A3